core/common/src/PlanarClipMask.ts

```diff
@@ -47,71 +42,85 @@ export enum PlanarClipMaskPriority {
 }
 
-/** Normalized representation of [[PlanarClipMaskProps]] with validation and default values applied.
- * @beta
+/** Describes how to mask the geometry of one [GeometricModel]($backend) for display. The mask is produced by projecting geometry from any number of other models -
+ * optionally filtered by [SubCategory]($backend) or by a set of specific [GeometricElement]($backend)s - onto a plane. Regions of the masked model that intersect the
+ * mask are rendered partially or completely transparent. This is useful for, e.g., making subsurface geometry visible below the background map, or clipping out portions
+ * of a reality model that intersect a design model.
+ * @note Currently reality models (including background maps and terrain) can be masked, but design models can only produce masks.
+ * @see [[DisplayStyleSettings.planarClipMasks]] to define clip masks for a [DisplayStyle]($backend).
+ * @see [ContextRealityModelState.planarClipMask]($frontend) to apply a clip mask to a context reality model.
+ * @public
  */
 export class PlanarClipMaskSettings {
-  /** The mode that controls how the mask geometry is collected */
-  public readonly mode: PlanarClipMaskMode = PlanarClipMaskMode.None;
-  /** The model IDs for the mask geometry.  If omitted then the models viewed in the masked model viewport are used */
+  /** Specifies how the mask geometry is produced. */
+  public readonly mode: PlanarClipMaskMode;
+  /** For any mode other than [[PlanarClipMaskMode.Priority]], the Ids of the [GeometricModel]($backend)s containing the geometry used to produce the mask.
+   * If `undefined`, the set of all models in the view's [ModelSelector]($backend) is used.
+   * The mask geometry can be filtered by [[subCategoryOrElementIds]].
+   */
   public readonly modelIds?: CompressedId64Set;
-  /** The SubCategory ids if mode is [[PlanarClipMaskMode.IncludeSubCategorie]] or element Ids if mode is [[PlanarClipMaskMode.IncludeElements] or [[PlanarClipMaskMode.ExcludeElements]] */
+  /** For [[PlanarClipMaskMode.IncludeElements]] or [[PlanarClipMaskMode.ExcludedElements]], the Ids of the [GeometricElement]($backend)s to include or exclude from masking;
+   * for [[PlanarClipMaskMode.IncludeSubCategories]], the Ids of the subcategories whose geometry contributes to the mask.
+   */
   public readonly subCategoryOrElementIds?: CompressedId64Set;
-  /** Priority value if mode is [[PlanarClipMaskMode.Priority]]. */
+  /** For [[PlanarClipMaskMode.Priority]], the priority value. */
   public readonly priority?: number;
-  /** A value between 0 and 1 indicating am override for mask transparency - A transparency of 0 (the default) indicates complete masking.  1 is completely transparent (no masking).  If no transparency is defined then the transparency of the mask elements are used */
+  /** A value between 0 and 1 indicating an override for mask transparency. A transparency of 0 (the default) indicates complete masking. 1 is completely transparent (no masking).
```

Comment:
@RBBentley it says zero is the default but also that `undefined` means "use element transparency". Only the latter is true, correct?

Reply:
Yes -- "(the default)" should be removed.